### ISSUE TITLE
accessing _cachedSystemAnimationFence requires the main thread (TiSDK 6.0.3)

### ISSUE THREAD

**Issue by timoa** [hansemannn/ti.onepassword#2]
Hi!

Thank you so much for this extension!

I try to integrate it but I get this error when I click on a button with OnePassword.findLoginForURLString:

```
[ERROR] Script Error {
[ERROR]     column = 3222;
[ERROR]     line = 1;
[ERROR]     message = "accessing _cachedSystemAnimationFence requires the main thread";
[ERROR]     sourceURL = "file:///var/containers/Bundle/Application/27845C27-6376-4110-A674-0DD693AB1CF5/******.app/alloy/controllers/Login.js";
[ERROR]     stack = "[native code]\nd@file:///var/containers/Bundle/Application/27845C27-6376-4110-A674-0DD693AB1CF5/******.app/alloy/controllers/Login.js:1:3222";
[ERROR] }
 ```

Any idea if it's related to the TiSDK (6.x) or maybe my integration?

I will try with a simple project and update this ticket soon.

Thank you for your help!

Damien

**Comment by hansemannn** [comment]
Thanks! You are probably running on kroll-thread. I'll update the library to be compatible with it as well.

**Comment by hansemannn** [comment]
Fixed as part of [v2.0.0](https://github.com/hansemannn/ti.onepassword/releases/tag/v2.0.0).
